include/rpt/particle_detector_interactions.h

```diff
@@ -57,6 +57,9 @@ class ParticleDetectorInteractions
     , detector_radius(detector.get_radius())
     , detector_length(detector.get_length())
     , parameters(rpt_parameters)
+    , dead_time(detector.dead_time)
```

Comment:
Add the description of these arguments after line 49 above. Every parameter must be detailed :)!